src/main/kotlin/com/openlattice/chronicle/services/timeusediary/TimeUseDiaryService.kt

```diff
@@ -153,7 +153,7 @@ class TimeUseDiaryService(
     ): Map<String, Set<String>> {
```

Comment:
We've been getting rid of multi-valued properties. Are these actually multivalued or is an array of one?

Reply:
It's actually multivalued. The map associates a question string to a set that contains the many responses to that question. 

Each entry in the map is then converted into its own column in the csv, where the key becomes the header and each response becomes a data in that column.

Reply:
Okay, I'm going to file this under as long as it works, but it sounds  a little crazy.